site/en/blog/new-in-devtools-83/index.md

```diff
@@ -44,7 +44,8 @@ differently (being able to perceive more/less of the relevant colors).
 By designing for the more extreme simulations in DevTools, your web apps are guaranteed to be
 accessible to people with protanomaly, deuteranomaly, tritanomaly, and achromatomaly as well.
 
-Send feedback to [Chromium issue #1003700][3].
+Send feedback to [Chromium issue #1003700][3], or [read more about the
+implementation](https://developer.chrome.com/blog/cvd/).
```

Comment:
```suggestion
implementation](https://developer.chrome.com/blog/cvd/).
```
```suggestion
implementation](/blog/cvd/).
```

Reply:
this will fix the linter issue.

Reply:
Thanks!

Random thought: this seems like the kind of thing that could potentially be stripped at build time, instead of warned about at lint time.